docs/bis/ec/update-koq-on-property.md

```diff
@@ -0,0 +1,186 @@
+# KindOfQuantity
+
+> Commonly referred to as a [KOQ](./kindofquantity.md).
+
+Describes a strongly typed _kind_ for a property. This kind identifies what is being measured and stored in an [ECProperty](./ec-property.md). It is used for grouping like properties, setting units and display formatting, and creating quantity objects.
+
+## Removing KindOfQuantity from a property
+
+Removing KOQ from an [ECProperty](./ec-property.md) is supported by using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) on the property. However, while using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) it must have both 'From' and 'To' properties set.
+
+Additionally, the 'From' value must match the persistence unit from the old KindOfQuantity and the 'To' value must be empty.
+
+### Examples
+
+Consider following [ECProperty](./ec-property.md) having [KindOfQuantity](./kindofquantity.md) attached to it:
+
+```xml
+<KindOfQuantity typeName='MyKindOfQuantity' description='My KindOfQuantity'
+				displayLabel='My KindOfQuantity' persistenceUnit='CM' relativeError='.5'
+				presentationUnits='FT;IN;M' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='MyKindOfQuantity' />
+</ECEntityClass>
+```
+
+Following are some Valid and Invalid ways of using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) to remove [KOQ](./kindofquantity.md) from the ECProperty:
+
+#### Valid `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md)
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:CM</From>
+			<To></To>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+#### Invalid/malformed `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md)
+
+```xml
+<ECProperty propertyName='Length' typeName='double' />
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:CM</From>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:M</From>
+			<To></To>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
```

Comment:
I don't think you need all these explicit examples.  Just briefly explain that From must be set to the source unit and To must be there but empty.